docs/export_emails.rst

```diff
@@ -86,8 +86,8 @@ There are a couple of settings keys which can be configured in `settings.py`.
 Below the default values are shown:
 
 ```
```

Comment:
This probably needs to be:
```
::

  EXPORT_EMAILS_ORDER_BY = ['last_name', 'first_name', 'username', 'email']
  EXPORT_EMAILS_FIELDS = ['last_name', 'first_name', 'username', 'email']
  EXPORT_EMAILS_FULL_NAME_FUNC = None
```

Instead of the markdown style ``` lines ?

Reply:
Could you change that ? and we can merge the PR ;-)

Reply:
Oh you are right! I'll make a PR right away